src/preset_cli/cli/superset/sync/dbt/datasets.py

```diff
@@ -86,11 +86,13 @@ def sync_datasets(  # pylint: disable=too-many-locals, too-many-branches, too-ma
 
         # load additional metadata from dbt model definition
         model_kwargs = model.get("meta", {}).pop("superset", {})
-        certification = (
-            model_kwargs.get("extra", {}).pop("certification")
-            if "certification" in model_kwargs
-            else certification or {"details": "This table is produced by dbt"}
-        )
+        certification_info = {
+            "certification": (
+                model_kwargs.get("extra", {}).pop("certification")
+                if "certification" in model_kwargs.get("extra", {})
+                else certification or {"details": "This table is produced by dbt"}
+            ),
+        }
```

Comment:
Just a nit, but sometimes it's cleaner to use a `try/except`:

```python
try:
    certification_info = model_kwargs["extra"]["certification"]
except KeyError:
    certification_info = certification or DEFAULT_CERTIFICATION
```

(and we'd define `DEFAULT_CERTIFICATION`)

Reply:
That's a very good feedback, @betodealmeida! I'm going to merge this PR for now to solve the issue in `main`, and address this in a future PR.

Reply:
Handled this here: https://github.com/preset-io/backend-sdk/pull/221